diff --git a/node_modules/vue-toastification/composition/index.js b/node_modules/vue-toastification/composition/index.js
index 021edc1..aa4ea1d 100644
--- a/node_modules/vue-toastification/composition/index.js
+++ b/node_modules/vue-toastification/composition/index.js
@@ -1,5 +1,5 @@
 const Vue = require("vue"); // eslint-disable-line @typescript-eslint/no-var-requires
-const { inject, provide } = require("@vue/composition-api"); // eslint-disable-line @typescript-eslint/no-var-requires
+const { inject, provide } = require("vue"); // eslint-disable-line @typescript-eslint/no-var-requires
 
 const toastSymbol = Symbol("Vue Toastification");
 
